docs/topics/feed-exports.rst

```diff
@@ -45,6 +45,7 @@ JSON lines
 ----------
 
  * Value for the ``format`` key in the :setting:`FEEDS` setting: ``jsonlines``
+   or ``jsonl``
```

Comment:
I have a slight preference towards leaving this as it was, even if we support `jsonl` and `jl` as aliases.

But, _if_ we add it here, we should also add `jl` for completion.

Reply:
+1 to keep it without alias.

Reply:
What's your reasons to keep just `jsonlines`?

Reply:
We *must* give the freedom to users to use whatever file extension they wish.
We *should* recognize file extensions of JSON Lines files, to use JSON Lines format automatically when users use any of those file extensions.
We *can* ask users to set `"format": "jsonlines"` if they want files with arbitrary extensions to use JSON Lines format.

I believe we happen to support `"format": "jl"` and `"format": "jsonl"` because that [makes our code simpler](https://github.com/scrapy/scrapy/blob/9f81de2a50568c3bfefc731d91f326ff14bcfd76/scrapy/utils/conf.py#L132) (otherwise we would need to split `FEED_EXPORTERS`, e.g. have a setting mapping MIME types to exporters, and a separate setting mapping file extensions to MIME types), not because we want to allow shorter, less-readable setting definitions.

So I would call the fact that we support `"format": "jl"` and `"format": "jsonl"` an implementation detail, and by not documenting it, it could be argued that we are free to drop support for it without backward-compatibility concerns if that makes sense in the future.